docs/developer-guide/code-overview.md

```diff
@@ -100,6 +100,12 @@ The e2e tests code is in [test](https://github.com/kubernetes/ingress-nginx/tree
 ## Other programs
 
 Describe here `kubectl plugin`, `dbg`, `waitshutdown` and cover the hack scripts.
+#### kubectl plugin
```

Comment:
You can add a blank line before this line

Reply:
Also I would suggest to use 3 hashes instead of 4